app/src/command.h

```diff
@@ -32,7 +32,7 @@
 #endif
 # define NO_EXIT_CODE -1
 
-process_t cmd_execute(const char *path, const char *const argv[]);
+int cmd_execute(const char *path, const char *const argv[], process_t *process);
```

Comment:
(and if the error message is printed directly in the specific implementation, there is no need to change this signature: `PROCESS_NONE` will report the error)